Automattic/wp-calypso
# Remove beta testing page for all users

#### Changes proposed in this Pull Request

Remove the beta testing page from the source code. This is blocked until Jetpack is released (scheduled date is 2021-11-02), following the discussion on https://github.com/Automattic/wp-calypso/pull/56847#issuecomment-939069133.

#### Testing instructions

Check that you cannot open `/beta-testing/%s` on Calypso, WPCOM, and Jetpack sites.



Resolves #56312.
